<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditWnd</class>
 <widget class="QWidget" name="EditWnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактирование значения</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QPushButton" name="bnCancel">
     <property name="text">
      <string>Отмена</string>
     </property>
     <property name="icon">
      <iconset resource="GenSv.qrc">
       <normaloff>:/GenSv/Resources/cancel.ico</normaloff>:/GenSv/Resources/cancel.ico</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLineEdit" name="leValue">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="bnSave">
     <property name="text">
      <string>Сохранить</string>
     </property>
     <property name="icon">
      <iconset resource="GenSv.qrc">
       <normaloff>:/GenSv/Resources/edit_button.ico</normaloff>:/GenSv/Resources/edit_button.ico</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="GenSv.qrc"/>
 </resources>
 <connections/>
</ui>
